src/Agent.Service/Windows/AgentService.csproj

```diff
@@ -57,15 +48,14 @@
       <DependentUpon>AgentService.cs</DependentUpon>
     </Compile>
     <Compile Include="Program.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Update="Properties\AssemblyInfo.cs" />
```

Comment:
I haven't seen Update attribute, is that a new thing in .NET core?

Reply:
Seems like they/ve added this attribute for .net core projects which are using [VS 2017+](https://learn.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild?view=vs-2022#attributes)